pyqtgraph/graphicsItems/ROI.py

```diff
@@ -1200,14 +1200,16 @@ class Handle(UIGraphicsItem):
     sigClicked = QtCore.Signal(object, object)   # self, event
     sigRemoveRequested = QtCore.Signal(object)   # self
     
-    def __init__(self, radius, typ=None, pen=(200, 200, 220), parent=None, deletable=False):
+    def __init__(self, radius, typ=None, pen=fn.mkPen(200, 200, 220), parent=None, deletable=False,
+                 activePen = fn.mkPen(255, 255,0)):
```

Comment:
Hei, I understand, but how would be the best way then to let the user choose different linewidths for the active and inactive state? Add a linewidth parameter for each?

I also have to apologize, those changes where not supposed to be in the pull request i did not know that all future commits also go into the pull request. The main change i wanted to propose where the position one in line 508.